x-pack/plugins/apm/public/hooks/use_fetcher.tsx

```diff
@@ -136,6 +138,7 @@ export function useFetcher<TReturn>(
   }, [
     counter,
     preservePreviousData,
+    rangeId,
```

Comment:
I just realised something, what happens if there's a fetch call that doesn't use the range, will it refresh anyway? I'd think that's fine anyway, but maybe good to think about.